packages/gatsby-theme-docs/src/components/code-example.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { CodeBlock, ContentNotifications } from '@commercetools-docs/ui-kit';
+import useCodeExamples from '../hooks/use-code-examples';
+
+function CodeExample(props) {
+  const codeExamples = useCodeExamples();
+
+  const codeExample = codeExamples.find(example => {
+    return example.absolutePath.includes(props.file);
+  });
+
+  if (!codeExample) {
+    return reportError('Code example does not exist');
```

Comment:
Let's provide a more useful error message

```suggestion
    return reportError(`Unable to find file "${props.file}" within the "code-examples" folder. Make sure that the file exists and matches the value passed to the "file" property.`);
```